src/phases/execution/MultiDeletes.yml

```diff
@@ -0,0 +1,162 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs inserts on one collection while a mass deletion takes place on another collection in the same database.
+
+GlobalDefaults:
+  # Configurable parameters
+  DocumentSizePaddingBytes: &DocumentSizePaddingBytes {^Parameter: {Name: "DocumentSizePaddingBytes", Default: 0}}
+  DocumentCount: &DocumentCount {^Parameter: {Name: "DocumentCount", Default: 100_000_000}}
+
+# Constant variables.
+WriteRate: &WriteRate 1 per 500 microseconds  # 2000/second
+Database: &Database test
+# Collection0 is the default collection for the MonotonicSingleLoader.
+MassDeletionsColl: &MassDeletionsColl Collection0
+
+Actors:
+
+# Phase 1: Drop the database.
+- Name: DropDatabase
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Populate the collection for mass deletion. TODO Make multithreaded once TIG-2938 is resolved.
+- Name: PopulateInitialData0
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *Database
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: *DocumentCount
+    BatchSize: 1000
+    Document:
+      padding: {^FastRandomString: {length: *DocumentSizePaddingBytes}}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Gaurd against timout due to no logging during PrePoulation0.
+- Name: LoggingActor0
+  Type: LoggingActor
+  Threads: 1
+  Phases:
+  - *Nop
+  - LogEvery: 9 minutes
+    Blocking: None
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 3: Issue a large delete that should take a considerable amount of time to run. This delete
+# will run with no concurrent writes to the database.
+- Name: LargeMultiDelete0
+  Type: CrudActor
+  Database: *Database
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    MetricsName: DuringMassDeletion0
+    Collection: *MassDeletionsColl
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: deleteMany
+          # Filter by the smallest OID so all documents are included.
+          Filter: {_id: {$gte: {^ObjectId: "000000000000000000000000"}}}
```

Comment:
We could filter by a new field "intToFilterBy" that is not_id. It could be randomly generated between 0 and DocumentCount, but there would be no uniqueness guarantee. It seems like setting the ObjectId to all 0's is sufficient.